tvtk/pyface/tvtk_scene.py

```diff
@@ -683,6 +684,10 @@ def save_gl2ps(self, file_name, exp=None):
                 self._exporter_write(ex)
             else:
                 ex.write()
+            # Work around for a bug in VTK where it saves the file as a
+            # .pdf.gz when the file is really a PDF file.
```

Comment:
Is this bug known to the VTK developers? If so, it would be useful to have a reference to a VTK issue, so that future readers of this code have a way to determine whether the upstream bug has been fixed or not.